jobs/build-arch.Jenkinsfile

```diff
@@ -161,7 +161,9 @@ lock(resource: "build-${params.STREAM}-${basearch}") {
         // We set the session to time out after 4h. This essentially
         // performs garbage collection on the remote if we fail to clean up.
         pipeutils.withPodmanRemoteArchBuilder(arch: basearch) {
-        def session = shwrapCapture("cosa remote-session create --image ${image} --expiration 4h")
+        def session = shwrapCapture("""
+        cosa remote-session create --image ${image} --expiration 4h --workdir ${env.WORKSPACE}
+        """)
```

Comment:
This feels like it should be a separate commit with its own motivation.

Reply:
broken out into a separate commit